test/Microsoft.AspNet.Razor.Test/Utils/TestFile.cs

```diff
@@ -27,7 +27,9 @@ public static TestFile Create(string localResourceName)
 
         public Stream OpenRead()
         {
-            var strm = Assembly.GetManifestResourceStream(ResourceName);
+            string resourceName = Assembly.GetName().Name + "." + ResourceName.Replace('/', '.');
```

Comment:
Actually looks like we don't set `ResourceName` anywhere. Make it `public string ResourceName { get; }` and do this calculation in the constructor:

```C#
public TestFile(string resName, Assembly asm)
{
    ResourceName = Assembly.GetName().Name + "." + resName.Replace('/', '.');
    Assembly = asm;
}

public string ResourceName { get; }

public Assembly Assembly { get; }
```